modules/migration-debugging-velero-admin-webhooks.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+[id="migration-debugging-velero-admin-webhooks_{context}"]
+= Issues with Velero and admission webhooks
+
+Velero has limited abilities to resolve admission webhook issues during a restore, so you might need to use an additional Velero plug-in or make changes to how you restore the workload.
+
+Typically workloads with admission webhooks require you to create a resource of a specific kind first.
+
+Documented restore workarounds:
+
+* link:https://github.com/knative/serving/issues/10926#issuecomment-886434177[Knative] requires you to resrore the top level `Service` resource be restored first by entering the following command:
```

Comment:
@RichardHoch 
"...requires you to res**t**ore the top level `Service` resource first..."

Reply:
@RichardHoch 
You've missed this one.

Reply:
Fixed